pokemongo_bot/step_walker.py

```diff
@@ -2,7 +2,7 @@
 
 from cell_workers.utils import distance
 from human_behaviour import random_lat_long_delta, sleep
-
+import random
```

Comment:
I prefer the flat `import random` because it makes using the functions like `random.uniform()` or `random.choice()` more obvious what they are.  Other files in this project do `from random import uniform` then `uniform()`